app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
You should use `find_by` for single record search in Rails 4+. It is more effective than `where(...).first` which was popular in Rails 3. (Videos are in Rails 3)